src/coreclr/src/pal/prebuilt/corerror/mscorurt.rc

```diff
@@ -314,14 +314,14 @@ BEGIN
 	MSG_FOR_URT_HR(CORDBG_E_UNSUPPORTED_DELEGATE) "The delegate contains a delegate currently not supported by the API."
 	MSG_FOR_URT_HR(PEFMT_E_64BIT) "File is PE32+."
 	MSG_FOR_URT_HR(PEFMT_E_32BIT) "File is PE32"
-	MSG_FOR_URT_HR(NGEN_E_SYS_ASM_NI_MISSING) "NGen cannot proceed because Mscorlib.dll does not have a native image"
+	MSG_FOR_URT_HR(NGEN_E_SYS_ASM_NI_MISSING) "NGen cannot proceed because System.Private.CoreLib.dll does not have a native image"
```

Comment:
Is "ngen" a term we want to present to customers in .NET Core? Shoudl these say crossgen?

Reply:
This string is unreachable in .NET Core. I am going to delete it.